core/controllers/admin.py

```diff
@@ -1101,7 +1101,7 @@ def get(self):
 class DeleteUserHandler(base.BaseHandler):
     """Handler for deleting a user with specific ID."""
 
-    @acl_decorators.can_access_admin_page
+    @acl_decorators.can_delete_any_user
```

Comment:
Can the old decorator now be removed?

Reply:
No, it is still used in other places.

Reply:
Ah, I missed that it's used elsewhere in this file (did a quick search earlier and only saw two matches--didn't look closely enough).